src/components/FilterSidebar/styles.scss

```diff
@@ -3,7 +3,7 @@
 .filter-sidebar {
   position: relative;
   width: 100%;
-  overflow-y: auto;
+  overflow: hidden auto;
```

Comment:
What is this for? I don't see any difference between this and the old style when running locally. This states you want no X overflow and Y to be auto, but I didn't see an X overflow.

Reply:
This is definitely still causing scrollbars to appear for me, I tested on various resolutions and they are appearing each time

Reply:
I'm satisfied then, leave it as you have it. I'm resolving this conversation.